### PR TITLE
add e2e-test arm64 support

### PR DESCRIPTION
**Pull request by cyclinder**
---
name: Pull request
about: Tell us about your contribution
labels: ["pr/release/none-required"]
---

---
## Thanks for contributing !

Before submitting a pull request, make sure you read about our Contribution notice here: <https://spidernet-io.github.io/spiderpool/contributing/development/pullrequest/#need-review>

**What this PR does / why we need it**:
add e2e-test  arm support
**Which issue(s) this PR fixes**:
None
**Special notes for your reviewer**:
`make clean` it doesn't work in mac, Since `xargs: illegal option -- i`
**make sure your commit is signed off**
Signed-off-by: cyclinder qifeng.guo@daocloud.io

### CODE REVIEW

File: test/Makefile.defs [hunk 1]
```diff
@@ -38,7 +38,7 @@ E2E_GINKGO_PROCS ?= 4
 
 #============Image version===========
 IMAGE_MULTUS_NAME := ghcr.io/k8snetworkplumbingwg/multus-cni:thick
-IMAGE_WHEREABOUTS_NAME := ghcr.io/k8snetworkplumbingwg/whereabouts:latest-amd64
+IMAGE_WHEREABOUTS_NAME := release-ci.daocloud.io/network-ci/k8snetworkplumbingwg/whereabouts:latest-$(ARCH)
```
Comment (weizhoublue) [inline test/Makefile.defs:41]: 开源项目，可能 得 避开 daocloud 的东西

Comment (cyclinder) [reply test/Makefile.defs:41]: 有点尴尬 目前whereabouts官方没提供arm的镜像 所以可能我们需要自己做然后再找个地方存

Comment (weizhoublue) [reply test/Makefile.defs:41]: 可以推一个到 spiderpool的repo下，或者 docker hub 是否有能找到下载源

Comment (cyclinder) [reply test/Makefile.defs:41]: docker hub我找过 都是个人repo下有arm版  推到spiderpool下比较合适点

**Comment by cyclinder** [comment]
whereabouts提了一个[PR](https://github.com/k8snetworkplumbingwg/whereabouts/pull/227), 可以构建并推送arm64的镜像。就不推到spider下，还是使用官方的，不然定期维护也麻烦，现在像multus等基本都提供amd和arm的多架构镜像

**Review by weizhoublue — APPROVED** [review]

**Pull request merged**